actions.yaml

```
@@ -7,11 +7,20 @@ get-primary:
 get-password:
   description: Change the admin user's password, which is used by charm.
     It is for internal charm users and SHOULD NOT be used by applications.
+  params:
+    username:
+      type: string
+      description: The username, the default value 'operator'.
+        Possible values - operator, pbmuser.
```

lets rename the user to `backup`
```suggestion
        Possible values - operator, backup.
```